starboard/nplb/player_test_util.cc

```diff
@@ -209,6 +178,37 @@ std::string GetSbPlayerTestConfigName(
   return name;
 }
 
+void SkipTestIfNotSupport(const SbPlayerTestConfig& config) {
+  SbMediaAudioCodec audio_codec = kSbMediaAudioCodecNone;
+  if (config.audio_filename && strlen(config.audio_filename) > 0) {
+    VideoDmpReader dmp_reader(config.audio_filename,
+                              VideoDmpReader::kEnableReadOnDemand);
+    SB_DCHECK(dmp_reader.number_of_audio_buffers() > 0);
+    if (!SbMediaCanPlayMimeAndKeySystem(dmp_reader.audio_mime_type().c_str(),
+                                        config.key_system)) {
+      GTEST_SKIP() << "Skipping tests due to unsupported audio config.";
```

Comment:
nit: Here and in the other GTEST_SKIP()s, you can remove the "Skipping tests due to", and just leave the "Unsupported bla" part, since the SKIP is already printed by GTest.